Library/Tracking/KSRAnalytics.swift

```diff
@@ -60,7 +60,7 @@ public final class KSRAnalytics {
     case signupSubmitButtonClicked = "Signup Submit Button Clicked"
     case skipVerificationButtonClicked = "Skip Verification Button Clicked"
     case tabBarClicked = "Tab Bar Clicked"
-    case thanksPageViewed = "Thanks Page Viewed"
+    case thanksPageViewed = "Page Viewed"
```

Comment:
It seems like we are moving toward a solution where the event names will be entirely generic like `pageViewed` and `ctaClicked`. Can you create a new enum below this one with `pageViewed` and delete the `thanksPageViewed` case from here?